packages/tenderly-core/src/internal/cli/commands/LoginCommand.ts

```diff
@@ -53,14 +63,20 @@ const validator = async function (value: string) {
 
 async function canAuthenticate(accessToken: string): Promise<boolean> {
   try {
+    logger.debug("Determining if the user can be authenticated with provided access token:", accessToken);
+
+    logger.trace("Making a call to the url...");
     const response = await axios.get(`${TENDERLY_API_BASE_URL}/api/v1/user`, {
       headers: { "x-access-key": accessToken },
     });
     if (response.data.user !== undefined) {
+      logger.debug("The user can be authenticated.");
       return true;
     }
+    logger.error("The user cannot be authenticated with access token:", accessToken);
     return false;
   } catch (err) {
+    logger.error(`There was an error while making the api call ${TENDERLY_API_BASE_URL}/api/v1/user`);
```

Comment:
I guess we are fine to expose `TENDERLY_API_BASE_URL` in the logs, right?

Reply:
Double check this, since we said we are not going to expose anything from the ENV vars in the logs!?!